pkgs/development/haskell-modules/lib.nix

```diff
@@ -259,6 +259,9 @@ rec {
    */
   buildStrictly = pkg: buildFromSdist (failOnAllWarnings pkg);
 
+  /* Disable core optimizations, significantly speeds up build time */
+  disableCoreOpts = pkg: appendConfigureFlags pkg ["--ghc-options=-O0"];
```

Comment:
Why not pass `--disable-optimization`?

Reply:
I wasn't aware it existed, what is the difference?

Reply:
One flag works for `ghc` only, the other one one works for all compilers that Cabal supports.

Reply:
>  Does nix have any Haskell compilers besides GHC?

Does it matter?

Reply:
@peti In principle I guess more can be added, I can change it, do you want me to change it?

Reply:
I've never seen that flag, it must not be documented.
```
 ghc --show-options | grep disable
```
Am I missing something?

Reply:
Yes, please.

Reply:
@peti it is finished, is it ok if we squash merge from the GitHub UI?

Reply:
cabal v1-configure --help

Reply:
@peti, thanks, always just done `ghc-options=-O0`.